TE-30191 Replace library django-cache-machine

Oleh-py opened the pull request:
# What?
 - remove six package to support django 3.2

# Why?
 - upgrade to django 3.2

SviatikKh commented on caching/invalidation.py:
```diff
@@ -10,8 +8,9 @@
-from django.utils import encoding, six, translation
-from django.utils.six.moves.urllib.parse import parse_qsl
+from django.utils import encoding, translation
+from django.utils import encoding,translation
```
should be deleted, probably duplicated

Oleh-py committed:
TE-30191 Replace library django-cache-machine

# What?
 - remove six package to support django 3.2

# Why?
 - upgrade to django 3.2

SviatikKh reviewed (approved):
Good job :1st_place_medal:
